README.md

```diff
@@ -33,8 +33,7 @@ WARNING: do not expose `--address` to the internet
 <b>NOTE: if the `--address` is not the same as your external IP addres used for communications between servers then you need to set `--real-ip`, otherwise the server will report internal IPs in the stats and it will run the test against itself, causing invalid results.</b>
 
 ### The listen command
-Hperf can run tests without a specific `client` needing to be constantly connected. Once the `client` has started a test, the `client` can 
-easily exit without interrupting the test stopping.
+Hperf can run tests without a specific `client` needing to be constantly connected. Once the `client` has started a test, the `client` can easily exit without interrupting the test stopping.
```

Comment:
But doesn't that make stats unreliable? Is this noted in stats?

Reply:
no, the test is completely ran by the server side binary. All the client does is send a config to initiate. 

Will refactor that sentence.

Reply:
Just asking, since I'd assume that missing a node would mean that you no longer track all the traffic... And the throughput would be lower because of that.

Reply:
The client doesn't run on a node.. it's just a control unit :) all the nodes remain connected.